specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-11-01-preview/sqlServerAvailabilityGroups.json

```diff
@@ -0,0 +1,802 @@
+      "get": {
+        "tags": [
+          "SqlServerAvailabilityGroups"
+        ],
+        "operationId": "SqlServerAvailabilityGroups_List",
+        "summary": "List the availability group associated with the given Arc Sql Server.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/sqlServerInstanceName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ArcSqlServerAvailabilityGroupListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***",
+            "schema": {
+              "$ref": "./common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets all availability groups associated with an Arc Enabled Sql server.": {
+            "$ref": "./examples/ListBySqlServerInstanceAvailabilityGroup.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AvailabilityGroupConfigure": {
+      "description": "The specifications of the availability group replica configuration",
+      "type": "object",
+      "properties": {
+        "endpointUrl": {
+          "type": "string",
+          "description": "Mirroring endpoint URL of availability group replica"
+        },
+        "availabilityModeDescription": {
+          "type": "string",
+          "description": "The Availability Synchronization mode of the availability group replica."
+        },
+        "failoverModeDescription": {
+          "type": "string",
+          "description": "The failover mode of the availability group replica."
+        },
+        "sessionTimeout": {
+          "type": "integer",
+          "description": "The time-out period of availability group session replica, in seconds.",
+          "format": "int32"
+        },
+        "primaryRoleAllowConnectionsDescription": {
+          "type": "string",
+          "description": "Whether the availability allows all connections or only read-write connections."
+        },
+        "secondaryRoleAllowConnectionsDescription": {
+          "type": "string",
+          "description": "Whether an availability replica that is performing the secondary role (that is, a secondary replica) can accept connections from clients."
+        },
+        "replicaCreateDate": {
```

Comment:
>replicaCreateDate [](http://example.com/codeflow?start=9&length=17)

Consider fix to follow naming guidelines:

☑️ YOU SHOULD use an "At" suffix in names of date-time values.

For example, createdAt not created or createdDateTime.